server/cors-settings.js

```diff
@@ -0,0 +1,5 @@
+var path = require('path')
+var fs = require('fs')
+
+var contents = fs.readFileSync(path.join(process.env['HOME'], '.binder/cors.conf'), 'utf8')
```

Comment:
Since you're using `path.join` here, you might as well make this `path.join(process.env['HOME'], '.binder', 'cors.conf')`